relay-server/src/actors/global_config.rs

```diff
@@ -0,0 +1,153 @@
+use std::sync::Arc;
+use std::time::Duration;
+
+use relay_dynamic_config::GlobalConfig;
+use relay_system::{Addr, AsyncResponse, FromMessage, Interface, Sender, Service};
+use tokio::sync::watch;
+
+use crate::actors::project_upstream::{GetProjectStates, GetProjectStatesResponse};
```

Comment:
It's not clean that we're using `project_upstream` types here, since we don't actually use that service. 
- Either create a minimal copy of the types that have just the fields we need
- Or move the types to another module

Especially that we're starting to use `pub(crate)` shows that we now get into strange API visibility. I'd heavily lean towards the first option and undo all changes to the upstream service file.